vars/RedHat.yml

```diff
@@ -2,3 +2,5 @@
 java: "{{ es_java | default('java-1.8.0-openjdk.x86_64') }}"
 default_file: "/etc/sysconfig/elasticsearch"
 es_home: "/usr/share/elasticsearch"
+package_manager: "{% if ansible_os_family == 'RedHat' and ansible_distribution_major_version >= '8' %}dnf{% else %}yum{% endif %}"
```

Comment:
Better use `ansible_distribution_major_version | int >= 8` because for higher version numbers this comparison is not returning the expected result, e.g. `'10' >= '8'` evaluates to `false`.

Reply:
Done